app/assets/stylesheets/heroic-features.scss

```diff
@@ -60,6 +60,10 @@ footer {
   color: black;
 }
 
+#profile {
+  color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.